src/tasks/processBulkUpload.ts

```diff
@@ -0,0 +1,15 @@
+import { getLogger } from '../logger';
+import { isProcessBulkUploadJobPayload } from '../types';
+
+const logger = getLogger(__filename);
+
+export const processBulkUpload = async (
+  payload: unknown,
+): Promise<void> => {
+  if (!isProcessBulkUploadJobPayload(payload)) {
+    logger.debug('Malformed bulk upload job payload');
+    logger.debug(isProcessBulkUploadJobPayload.errors);
+    throw new Error('The bulk upload job payload is not properly formed');
```

Comment:
For the most part, the Codecov checks in this PR feel a bit over the top, but I do agree with it here - we should have some tests for this task!